src/reactions/aggregateReactions.js

```diff
@@ -15,14 +15,18 @@ export default (
         code: x.emoji_code,
         count: 0,
         selfReacted: false,
+        users: [],
       };
       reactionMap.set(x.emoji_name, item);
     }
 
     item.count += 1;
+    item.users.push(x.user_id);
     if (x.user_id === ownUserId) {
       item.selfReacted = true;
     }
   });
-  return Array.from(reactionMap.values());
+  return Array.from(reactionMap.values()).sort(
+    (r1: AggregatedReaction, r2: AggregatedReaction) => r2.count - r1.count,
+  );
```

Comment:
This affects the reactions shown in the message list, too, right?

Does this match how the webapp orders these?

Whatever change we make here, probably best put at the start of the branch (in particular before creating the new screen) -- that way the screen looks the way we want it to from the beginning.